mmv1/third_party/terraform/services/networkconnectivity/resource_network_connectivity_hub_test.go

```diff
@@ -0,0 +1,146 @@
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"project_name":  envvar.GetTestProjectFromEnv(),
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckNetworkConnectivityHubDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccNetworkConnectivityHub_BasicHubLongForm(context),
+			},
+			{
+				ResourceName:            "google_network_connectivity_hub.primary",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"labels", "terraform_labels"},
+			},
+			{
+				Config: testAccNetworkConnectivityHub_BasicHubLongFormUpdate0(context),
+			},
+			{
+				ResourceName:            "google_network_connectivity_hub.primary",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"labels", "terraform_labels"},
+			},
+		},
+	})
+}
+func TestAccNetworkConnectivityHub_BasicHub(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"project_name":  envvar.GetTestProjectFromEnv(),
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckNetworkConnectivityHubDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccNetworkConnectivityHub_BasicHub(context),
+			},
+			{
+				ResourceName:            "google_network_connectivity_hub.primary",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"labels", "terraform_labels"},
+			},
+			{
+				Config: testAccNetworkConnectivityHub_BasicHubUpdate0(context),
+			},
+			{
+				ResourceName:            "google_network_connectivity_hub.primary",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"labels", "terraform_labels"},
+			},
+		},
+	})
+}
+
+func testAccNetworkConnectivityHub_BasicHubLongForm(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_network_connectivity_hub" "primary" {
+  name        = "long/form/tf-test-hub%{random_suffix}"
+  description = "A sample hub"
+  project     = "long/form/%{project_name}"
+
+  labels = {
+    label-one = "value-one"
+  }
+}
+
+
+`, context)
+}
+
+func testAccNetworkConnectivityHub_BasicHubLongFormUpdate0(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_network_connectivity_hub" "primary" {
+  name        = "long/form/tf-test-hub%{random_suffix}"
+  description = "An updated sample hub"
+  project     = "long/form/%{project_name}"
```

Comment:
```suggestion
  project     = "projects/%{project_name}"
```

Change this to projects/ as some things depend on it. These should fix the test failures